.github/workflows/verify_and_publish.yml

```diff
@@ -125,9 +125,16 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
 
+    - name: Install packages
```

Comment:
I don't think you need this anymore as it should be readily available in the docker image.